src/facts/meadow-blazingstar.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Meadow Blazingstar
+image:
+  src: ../images/15120801615_9334187129_h.jpg
+  alt: Meadow Blazingstar
+  creator: Photo by USFWS
```

Comment:
Right now this would read "Meadow Blazingstar image by Photo by USFWS" so we don't need the "Photo by" part. 
```suggestion
  creator: USFWS Midwest Region
```